### ISSUE TITLE
Kiosk: There should be a progress indicator while loading an app dynamically

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2730]
particularly Q&V is showing a bit of a blank page for too long for my taste.

**Comment by urapadmin** [comment]
### Kiosk 1.6.1
- dynamically loaded apps (currently only Q&V) have an initial progress indicator.

**Comment by luizaogs** [comment]
Is that the little rainbow? I see it in the hub, too.

**Comment by urapadmin** [comment]
ja, but in Q&V you see two :o) First the one I was missing and then there is a very short witch to a second one and that is the one the Hub has, too. Doesn't matter. The point is: You should not have to stare at the screen and nothing seems to be happening for more than a second.

**Comment by luizaogs** [comment]
Yup, it’s working.

**Comment by urapadmin** [comment]
reopening this because the rainbow animation is strangely short often enough. Particularly seeing this with Q&V
